node/node.go

```diff
@@ -7,6 +7,8 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
+	libp2p_peer "github.com/libp2p/go-libp2p-core/peer"
+
```

Comment:
Is it desired that we leave one line clear between lib code outside `harmony-one`?

Reply:
We do not have a team-level consensus yet; I personally prefer/use the stdlib-external-local grouping order, because that way the same-package imports are easier for human eyes to identify especially when dealing with import cycles and so on.

Reply:
I see, I prefer otherwise, no line breaks. Let's pick one way